src/SourceBuild/patches/aspnetcore/0001-fix-System.IdentityModel-version.patch

```diff
@@ -0,0 +1,28 @@
+From ee7a37e0e470dc3e926d13d8532a56cfd66763b2 Mon Sep 17 00:00:00 2001
+From: Oleksandr Didyk <odidyk@microsoft.com>
+Date: Mon, 24 Jul 2023 11:05:43 +0000
+Subject: [PATCH] fix System.IdentityModel version
+
+Backport: https://github.com/dotnet/aspnetcore/issues/49596
+---
+ eng/Baseline.Designer.props | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/eng/Baseline.Designer.props b/eng/Baseline.Designer.props
+index c9d8dfddfc..109cd4ec04 100644
+--- a/eng/Baseline.Designer.props
++++ b/eng/Baseline.Designer.props
+@@ -137,7 +137,9 @@
+   </PropertyGroup>
+   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.Authentication.WsFederation' AND ('$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)' OR '$(TargetFramework)' == 'net7.0') ">
+     <BaselinePackageReference Include="Microsoft.IdentityModel.Protocols.WsFederation" Version="[6.15.1, )" />
+-    <BaselinePackageReference Include="System.IdentityModel.Tokens.Jwt" Version="[6.21.0, )" />
++    // In source-build context this version is coming from source-build-externals and must be set to 6.21.0
```

Comment:
This is not valid comment syntax in msbuild - it should use <!-- -->

Reply:
That's on me, I added the comment after building and auto-piloted to `//`. Will fix